x/auth/client/cli/tx_sign.go

```diff
@@ -188,8 +183,8 @@ be generated via the 'multisign' command.
 	}
 
 	cmd.Flags().String(flagMultisig, "", "Address of the multisig account on behalf of which the transaction shall be signed")
-	cmd.Flags().Bool(flagAppend, true, "Append the signature to the existing ones. If disabled, old signatures would be overwritten. Ignored if --multisig is on")
-	cmd.Flags().Bool(flagSigOnly, false, "Print only the generated signature, then exit")
+	cmd.Flags().Bool(flagOverwrite, false, "Overwrite existing signatures with a new one. If disabled, new signature will be appended")
```

Comment:
I think we should be explicit that default behavior is 
```suggestion
	cmd.Flags().Bool(flagOverwrite, false, "Overwrite existing signatures with a new one. If false, new signature will be appended (default is false). Ignored if --multisig is on")
```
I think we should keep the expected behavior of ignoring if `--multisig` is on, and make sure the code reflects that as well.

Reply:
> I think we should keep the expected behavior of ignoring if --multisig is on

We don't have to enforce that. It was previously ignored, because the code didn't support it (it was assuming old signatures, not `SignatureV2`). 

Boolean flags should be defined / named in a way to have default=false (adding a flag should imply changing default behavior). So, `--overwrite=false` is default, which is good. Then if user want's to change default, he will type: `--overwrite` instead of `--overwrite=true`. He can also be explicit with default if he wants by writing `--overwrite=false`.

Reply:
This way we don't need to mention default for each command. If you want, But if you prefer, I can describe defaults of each flag in this command (or all commands in this file).

Reply:
No i think what you describe here is fine. So `--overwrite` flag would not be ignored then in the case of multisig transactions. Sure, i can imagine some users maybe wanting this, so let's leave it as is.

Reply:
> This way we don't need to mention default for each command.

Yeah I agree, it's extra work. Extra comments about combination of flags should be described in the cmd's long description IMHO.